.travis.yml

```diff
@@ -235,6 +235,7 @@ before_install:
   - chmod +x docker-compose
   - sudo mv docker-compose /usr/local/bin
   - if [ $TRAVIS_OS_NAME = osx ]; then pip install virtualenv==16.7.9; fi
+  - if [ $TRAVIS_OS_NAME = osx ]; then ulimit -n 1024; fi
```

Comment:
Was this added because "too many open files" error in travis? I think this will be solved by https://github.com/elastic/beats/pull/16681

Reply:
Yes, precisely. I think I can revert it here.